### DISCUSSION TITLE
[Feature Request]: 希望编辑订阅的时候可以像旧版nastool一样手动录入年份

### DISCUSSION THREAD

**Discussion by XR-LM** [jxxghp/MoviePilot#3408] [讨论]
### 当前程序版本

v1.5.0

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

![image](https://github.com/jxxghp/MoviePilot/assets/40235159/77ee243e-c773-4b08-8b28-343ad258f8c4)
目前年份不一致无法正确检索到资源
![image](https://github.com/jxxghp/MoviePilot/assets/40235159/08565b9c-e7f2-4c31-89cc-d37f225bf3be)

**Comment by iamckun** [comment]
我也遇到了吞噬星空有的站点分季了没有总的集数，直接没法用，只能手动去站点下载并给标签，整理一点毛病没有，就是订阅用不起来

**Comment by XR-LM** [comment]
自定义关键词替换后就可以正常订阅了
![image](https://github.com/jxxghp/MoviePilot/assets/40235159/f7fbd897-0c5d-4285-a962-f209e02a9de2)
